### ISSUE TITLE
Help adding potential talk topics

### ISSUE THREAD

**Issue by Cheukting** [ossf/DevRel-community#11]
Using the [template](https://github.com/ossf/DevRel-community/blob/main/abstracts/TALK_TEMPLATE.md) to create more talk topics that folks can use to give OSS security-related talks at various events.

All suggested topics can be put [here](https://github.com/ossf/DevRel-community/tree/main/abstracts).

**Comment by edodusi** [comment]
@Cheukting you mean using the template to create abstract talks to be put in that folder, as references to use for actual submissions to confs?

**Comment by Cheukting** [comment]
Yes @edodusi, the goal of creating those is to have something that folks can start using to submit for CfPs

**Comment by tabathad** [comment]
superseded by https://github.com/ossf/DevRel-community/issues/43
